src/Microsoft.Health.Fhir.Tests.Common/Microsoft.Health.Fhir.Tests.Common.csproj

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <IsPackable>true</IsPackable>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Adding this since dotnet pack(with the updgraded aspnetcore)is failing due to the errors which are also seen in the success dotnet packs for all other PRs with older sdk versions but somehow these errors were ignored and dotnet pack was never failed.
error NU5100: The assembly 'content\testhost.dll' is not inside the 'lib' folder and hence it won't be added as a reference when the package is installed into a project. Move it into the 'lib' folder if it needs to be referenced. rror NU5100: The assembly 'contentFiles\any\netcoreapp3.1\testhost.dll' is not inside the 'lib' folder and hence it won't be added as a reference when the package is installed into a project. Move it into the 'lib' folder if it needs to be referenced.

Reply:
It looks like the most likely reason for this is that the common project has a reference that I don't think it needs. I'm testing the build locally with the removal and will push an update shortly if it works.

Just for reference, the issue seems to be including `Microsoft.NET.Test.Sdk` as a reference in a non test project.

Reply:
Upon further investigation, it looks like this may be the culprit - https://github.com/NuGet/Home/issues/8867#issuecomment-569765203

By adding the property `<IncludeContentInPack>false</IncludeContentInPack>` to the test csproj projects I was able to no longer get the exception, but the downstream dependencies will need to be analyzed before making this change.

cc: @brendankowitz

Reply:
I'd be in favor of removing `Microsoft.NET.Test.Sdk` first to see if this works

Reply:
Microsoft.NET.Test.Sdk can be removed from Tests.Common project and packs fine. 
But for e2e test, I am seeing similar testhost.dll errors. As dicussed in Engineering hours meeting with Jack and Brandon, I will add the property <IncludeContentInPack>false</IncludeContentInPack> for e2e tests. And Pack e2e tests and test locally by referring in the Paas.